src/css/profile/mobile/theme-changeable/theme.color.less

```diff
@@ -268,14 +268,12 @@
 @color_dropdownmenu_list_group_stroke: B0733L1; // #[color] list stroke line color
```

Comment:
You can check and remove rest of unused variables for dropdown menu.

Reply:
Removed more variables.